pre_commit_hooks/requirements_txt_fixer.py

```diff
@@ -10,6 +10,19 @@ def __init__(self):
         self.value = None
         self.comments = []
 
+    @property
+    def name(self):
+        if self.value is None:
+            return
+
+        if self.value == b'\n':
+            return
+
```

Comment:
I think (and I believe the travis-ci output confirms) that the code from here up is dead (likely handled by the explicit checks in `__lt__` below)